src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import GearSelection from './components/gearSelection'
+import GearSelection from './components/gearGrid'
```

Comment:
**Tech Debt** We need to make this consistent whatever we decide, it is imported as `GearSelection` but from `gearGrid`. This can be tech debt for now but something we should fix.

Reply:
Will in the next commit